latexindent.pl

```diff
@@ -269,9 +269,32 @@
 
 # get information about user settings- first check if indentconfig.yaml exists
 my $indentconfig = File::HomeDir->my_home . "/indentconfig.yaml";
+
+if ( -e $indentconfig)
+{
+      print $logfile "Using settings file $indentconfig\n";
+      if ( -e File::HomeDir->my_home . "/.indentconfig.yaml"){
+        print $logfile File::HomeDir->my_home,"/.indentconfig.yaml has been found, but $indentconfig takes priority\n";
+      }
+      else
+      {
+        print $logfile "Alternatively, ",File::HomeDir->my_home,"/.indentconfig.yaml can be used\n";
+      }
+}
+else
+{
+      $indentconfig = File::HomeDir->my_home . "/.indentconfig.yaml";
+      if ( -e $indentconfig)
+      {
+        print $logfile "Using settings file $indentconfig\n";
+        print $logfile "Alternatively, ",File::HomeDir->my_home,"/indentconfig.yaml can be used\n";
+      }
+}
+
 if ( -e $indentconfig and !$onlyDefault )
 {
-      print $logfile "Reading path information from ",File::HomeDir->my_home,"/indentconfig.yaml\n";
+      print $logfile "Reading path information from $indentconfig\n";
+      print $logfile "Only default settings requested, not reading USER settings from $indentconfig\n";
```

Comment:
I don't think you wanted this line here :) This should only be used if `$onlyDefault` is used; can you delete it?